sunpy/net/tests/test_vso.py

```diff
@@ -475,3 +475,15 @@ def test_vso_error(client):
         client.search(
             va.Time('2019/12/30', '2019/12/31'),
             va.Instrument('ovsa'))
+
+
+@pytest.mark.remote_data
+def test_incorrect_content_disposition(client):
+    results = client.search(
+        va.Time('2011/1/1 01:00', '2011/1/1 01:02'),
+        va.Instrument('mdi'))
+    files = client.fetch(results[0:1])
+
+    assert len(files) == 1
+    assert  files[0].endswith("mdi_vw_v_9466622_9466622.tar")
```

Comment:
Not necessarily an issue with this PR but that file looks wrong (the name and tar extension). When I download it and expand the tar it contains a file called `error_list.txt` and this has the following entry
```
record = mdi.vw_V[2011.01.01_01:02:00_TAI], file = mdi.vw_V.2011.01.01_01:02:00_TAI.data.tas, message = cannot export Rice-compressed floating-point images
```

Reply:
I have opened https://github.com/sunpy/sunpy/issues/3747 to track this and have asked the VSO team about it.